CHANGES.rst

```diff
@@ -35,6 +35,12 @@ Features added
 * #11981: Improve rendering of signatures using ``slice`` syntax,
   e.g., ``def foo(arg: np.float64[:,:]) -> None: ...``.
 
+* The manpage builder now emits `OSC 8 hyperlink` anchors, via
+  the `groff device control command`.
+
+  .. `OSC 8 hyperlink`: https://gist.github.com/egmontkob/eb114294efbcd5adb1944c9f3cb5feda
```

Comment:
I assume that's the most "official" page for that right?

Reply:
yep, as odd as it looks, this is the canonical one